client/dist/style.css

```diff
@@ -1,2 +1,61 @@
+body {
+  margin: 0;
+  padding: 0;
+  background: #eee;
+}
+
 #app {
+  padding: 20px;
+}
+
+.module {
```

Comment:
Might not be a good idea to have a class called module just in case other people will have the same class when we are combining all our code into the same place. Something more descriptive like graph-module could be a good idea.

Reply:
@seungprk

Reply:
Agreed and fixed!